Templates/HttpTrigger-Powershell/run.ps1

```diff
@@ -0,0 +1,8 @@
+$name = Get-Content $Env:req
```

Comment:
Well the templates aren't really consistent - e.g. the BAT template only processes the name from the query string, otherwise returns "Enter a name"